js/layout-javascript/agenda-code.js

```diff
@@ -601,8 +601,6 @@ DynamicList.prototype.attachObservers = function() {
           return;
         }
 
-        _this.$container.find('.new-agenda-list-container, .dynamic-list-add-item').addClass('hidden');
```

Comment:
What's this for and why is it necessary for this feature? I see there's a lot of code changes in all the layouts where you're showing/hiding elements on the page before an entry is opened. I don't understand why this is related to the requirements of this ticket.

Reply:
I apologize for the delay. This function was not written by me and was bundled along with the previous changes. Would it be better to undo someone else's changes?

Reply:
![Image 2021-07-28 at 10 51 17 am](https://user-images.githubusercontent.com/290733/127302423-ed5bf347-1766-4317-a2a7-64a26483d3cb.jpg)

@skrupnyk If you go to the **Files changed** tab (see above), you can see all the code changes for this PR. You should know why every single line of the changes are necessary. If not, you need to find out/clarify/remove the changes.